### PR TITLE
Add total attendance column to workshop dashboard for local summer workshops

### PR DESCRIPTION
**Pull request by clareconstantine**
This is part of the work for adding the scholarship dropdown - it will help partners decide which teachers will receive scholarships.

For each enrollee, display the number of sessions they have attended / the total number of sessions the workshop has.

screen shot 2019-01-30 at 4 41 17 pm

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import {workshopEnrollmentStyles as styles} from "../workshop_enrollment_styles"
 
 const CSF = "CS Fundamentals";
 const NA = "N/A";
+const LOCAL_SUMMER = "5-day Summer";
```
Comment (islemaster) [inline apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx]: Can we / should we be sharing these constants with the server code?

**Review by islemaster — COMMENTED** [review]
Changes look correct. Can we get test coverage? I was thinking:

- Assert that `attendances` is included in the serialized workshop enrollment
- Assert that the Total Attendance column only shows up for Local Summer workshops

**Review by bethanyaconnor — APPROVED** [review]

File: apps/Gruntfile.js [hunk 2]
```diff
@@ -8,25 +8,32 @@ var logBuildTimes = require('./script/log-build-times');
 var webpackConfig = require('./webpack');
```
Comment (islemaster) [inline apps/Gruntfile.js]: Did you actually change Gruntfile.js or is this just a Prettier automated change?

File: apps/package.json [hunk 3]
```diff
@@ -153,7 +153,7 @@
     "pepjs": "^0.4.3",
     "phantomjs-prebuilt": "^2.1.14",
     "playground-io": "code-dot-org/playground-io#v0.6.0-cdo.0",
-    "prettier": "1.16.1",
+    "prettier": "1.16.4",
```
Comment (islemaster) [inline apps/package.json]: This also seems unrelated?

File: apps/Gruntfile.js (hunk 2 above)
Comment (clareconstantine) [reply apps/Gruntfile.js]: I didn't update the gruntfile at all. Should I commit again undoing the Prettier changes?

File: apps/package.json (hunk 3 above)
Comment (clareconstantine) [reply apps/package.json]: yep, also not intentional

File: apps/src/code-studio/soundLibrary.json [hunk 4]
```diff
@@ -3724,4 +3724,4 @@
       "category_male_voiceover/you_win_male"
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (islemaster) [inline apps/src/code-studio/soundLibrary.json]: It'd be nice to get `soundLibrary.json` entirely out of this PR, but understandable if that's tricky at this point.

File: apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_infoTest.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        numSessions={5}
+      />,
+      {context}
+    );
+
+    expect(
+      workshopEnrollmentSchoolInfo
+        .find('th')
+        .filterWhere(col => col.text().includes('Total Attendance'))
+    ).to.have.length(0);
```
Comment (islemaster) [inline apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_infoTest.js:55]: :+1: I dig it.  Thanks for adding additional coverage here!

**Review by islemaster — APPROVED** [review]
